homeassistant/components/hisense_aehw4a1/climate.py

```diff
@@ -158,7 +158,7 @@ def __init__(self, device):
         self._fan_modes = FAN_MODES
         self._swing_modes = SWING_MODES
         self._preset_modes = PRESET_MODES
-        self._available = None
+        self._attr_available = False
```

Comment:
`available` should never be `None`, so I think it was a bug.
I have adjusted this to use default `False`